fawltydeps/main.py

```diff
@@ -82,7 +82,8 @@ def resolved_deps(self) -> Dict[str, Package]:
         """The resolved mapping of dependency names to provided import names."""
         return resolve_dependencies(
             (dep.name for dep in self.declared_deps),
-            custom_mapping_path=self.settings.custom_mapping,
+            custom_mapping_path=self.settings.custom_mapping_file,
```

Comment:
Why not change to `custom_mapping_path` since this name is already used at different places?

Reply:
I think the change is good for consistency, but rather to `custom_mapping_file`. We do not do any traversing here, and `_path` suffix indicates it may be file or directory.